test/python/test_docker_pack.py

```diff
@@ -228,14 +233,15 @@ def test_base_dockerfile_with_env_vars(project_without_dependencies, module_tmpd
     cmd = [
         os.path.join(basepath, "cartridge"),
         "pack", "docker",
-        "--from", dockerfile_with_env,
+        "--from", dockerfile_with_env_path,
         project_without_dependencies.path,
     ]
-    process = subprocess.run(cmd, cwd=module_tmpdir)
-    assert process.returncode == 0
+    rc, output = run_command_and_get_output(cmd, cwd=module_tmpdir)
+    assert rc == 0
+    assert 'Detected base Dockerfile {}'.format(dockerfile_with_env_path) in output
 
 
-def test_docker_e2e(docker_image, tmpdir, docker_client):
+def test_e2e(docker_image, tmpdir, docker_client):
```

Comment:
but why?

Reply:
To get rid of `test_docker_pack.py::test_docker_pack` and `test_docker_pack.py::test_docker_e2e`.

I sure that `test_docker_pack.py::test_pack` and `test_docker_pack.py::test_e2e` are better.

Reply:
Ok, agree